src/layer/marker/index.js

```diff
@@ -2,6 +2,48 @@ import {Icon} from './Icon.js';
 export {icon} from './Icon.js';
 import {IconDefault} from './Icon.Default.js';
 Icon.Default = IconDefault;
+
+for (const color of ['Pink', 'DarkBlue', 'Lila', 'Magenta', 'Red', 'Orange', 'Brown', 'Green', 'Mint']) {
+	Icon.Default[color] = IconDefault.extend({
+		options:{
+			className: `default-icon-color-${color.toLowerCase()}`
+		}
+	});
+}
+Icon.Default.Blue = IconDefault;
+
+/**
+ * @miniclass Colored default icons (Icon)
+ * @section
+ *
+ * `L.Icon.Default` has several static properties providing different coloured versions of the default icon.
+ *
+ * Usage is as: `L.marker(latlng, { icon: new L.Icon.Default.Green() })`
+ *
+ * Note that `L.Icon.Default.Blue` behaves the same as `L.Icon.Default`.
+ *
+ * @property Pink: L.Icon
```

Comment:
Or is the class `L.Icon.Default`?

Reply:
That'd be correct, but I'd find that somewhat confusing, TBH. There's too many `L.Icon.Default` in that bit of docstring.